### PR TITLE
Do some cleaning and update PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by fabschurt**

### CODE REVIEW

File: php-cs-fixer/v2.13/Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN phive install \
 
 FROM php:7.2 as php-cs-fixer
 
-COPY --from=base /root/.phive/phars/php-cs-fixer.phar /usr/local/bin/php-cs-fixer
+COPY --from=base /home/docker/tools/php-cs-fixer /usr/local/bin/php-cs-fixer
```
Comment (fabschurt) [inline php-cs-fixer/v2.13/Dockerfile:13]: `/home/docker/tools/php-cs-fixer` est un symlink créé par Phive.

File: README.md [hunk 2]
```diff
@@ -6,13 +6,17 @@ Images:
 
 ## For maintainers
 
+**Note:** at the time of writing (October 2019), you’ll need to enable Docker’s
+[experimental features](https://github.com/docker/docker-ce/blob/master/components/cli/experimental/README.md)
+to be able to build the images.
```
Comment (nm2107) [inline README.md]: can you indicate what are the experimental features in use ?

Comment (fabschurt) [reply README.md]: :+1:

Comment (akerouanton) [reply README.md]: mh I don't see any experimental features in use :thinking:

Comment (fabschurt) [reply README.md]: @NiR-

```
./bin/build
"--squash" is only supported on a Docker daemon with experimental features enabled
make: *** [Makefile:3: build] Error 1
```

Comment (fabschurt) [reply README.md]:
```
$ docker --version
Docker version 19.03.3-ce, build a872fc2f86
```

Comment (akerouanton) [reply README.md]: :ah: Indeed I thought you added some but couldn't see which (and my client and daemon are configured to build with buildx by default) :sweat_smile:

Comment (fabschurt) [reply README.md]: ¯\\\_(ツ)_/¯

**Review by akerouanton — APPROVED** [review]

**Review by nm2107 — APPROVED** [review]
thank you :)

**Pull request merged**